interview-questions.md

```diff
@@ -4,49 +4,62 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer:
+  Your answer: Create/React/Update/Delete
 
-  Researched answer:
+  Researched answer: When we are building APIs, we want our models to provide four basic types of functionality. 
+  The model must be able to Create, Read, Update, and Delete resources. A model should have the ability to perform at most these four functions in order to be complete. If an action cannot be described by one of these four operations, 
+  then it should potentially be a model of its own.
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer:
+  Your answer: Post, Get, Delete
 
-  Researched answer:
+  Researched answer: Post, Get, Put, Patch, Delete
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer:
+  Your answer: "class HelloWorld extends React.Component" at the top, render and inside return "Hello World"
 
-  Researched answer:
+  Researched answer:class HelloWorld extends React.Component { 
+  render() {
+    return <h1>Hello, {this.props.name}</h1>;  
+  }
+}
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow